perma_web/api/tests/test_link_resource.py

```diff
@@ -684,7 +684,7 @@ def test_media_capture_in_iframes(self):
             # test_media_b.html
             ("test.mp4", "video/mp4"), ("test2.mp4", "video/mp4"),
             # test_media_c.html
-            ("test.swf", "application/x-shockwave-flash"), ("test2.swf", "application/x-shockwave-flash"), ("test3.swf", "application/x-shockwave-flash"),
+            ("test.swf", "application/vnd.adobe.flash.movie"), ("test2.swf", "application/vnd.adobe.flash.movie"), ("test3.swf", "application/vnd.adobe.flash.movie"),
```

Comment:
The MIME type for .swf files was changed in bullseye; see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=985039 for some discussion.